### PR TITLE
Scraping and data gathering

### LINKED ISSUE #59: Describe data gathering

### LINKED ISSUE #60: Describe scraping

### PR DESCRIPTION
**Pull request by finloop**
Closes #59 
Closes #60

### CODE REVIEW

File: presentation.md [hunk 1]
```diff
@@ -7,5 +7,45 @@ abstract: |
...
+Na szczęście udało nam się znaleźć inne źródła, tj. strony tworzone przez ludzi,
+którzy codziennie sami zbierali te dane i udostępniali je w formie wykresów.
+Przykładem takiej strony jest koronawirusunas.pl . Dzięki danym na niej zawartym
+udało nam się przeprowadzić.
+
+Do głównych źródeł danych zaliczyć możemy:
+
+- IHME
+- koronawirusunas.pl
+- policja.pl
```
Comment (pgronkievitz) [inline presentation.md]: You can split slides with 
```md
Lorem

---

ipsum
```
This slide is a bit too long

File: presentation.md [hunk 2]
```diff
@@ -7,5 +7,45 @@ abstract: |
...
+danych. Do tego wykorzystaliśmy pythona i Jupyterlab.
+
+Z pomoca jupyterlab można stworzyć interaktywne notatniki, które działają krok
+po kroku, jeden z nich służył wyłącznie do pobrania danych ze wszystkich źródeł
+ich oczyszczenia i sprowadzenia do wspólnego formatu. Same analizy znalazły się
+w osobnym notatniku, co umożliwiło nam zapanowanie nad projektem. Oczywiście
+korzystanie z notatników też ma swoje wady, kolejne zmiany utrzymywane w
+systemie kontroli wersji Git nie są czytalne.
+
+![](img/zar.png)
```
Comment (pgronkievitz) [inline presentation.md]: Same as above

Comment (finloop) [reply presentation.md]: Done.

File: presentation.md (hunk 1 above)
Comment (finloop) [reply presentation.md]: Done.

File: presentation.md [hunk 3]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
 ---
 
-# Źródła danych
+# O nas
+
+Jesteśmy studentami Politechniki Rzeszowskiej na kierunku Inżynieria i analiza
+danych, a także członkami nowo stworzonego koła uczenia maszynowego.
+[//]: # (Dodać logo koła naukowego)
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 4]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+było znalezienie, reguł asocjacyjnych i ciekawych zależności w danych z
+zachowań, które docelowo miały nam posłużyć do lepszego zrozumienia sposobu
+rozprzestrzeniania się koronawirusa Sars-Cov-2 i przewidywania ilości
+zachorowań.
+
+Pokażemy, w jaki sposób pracowaliśmy, jakich technik używaliśmy i jak zdobyliśmy
+dane, których używaliśmy. Mamy nadzieję, że są wśród was studenci, których
+zainteresujemy naszą pracą i dołączą do naszego koła naukowego.
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 5]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+
+# Dane
+
+Jeszcze, gdy zaczynaliśmy, liczba zachorowań nie przekraczała 500 osób
+dziennie (był to okres od marca do maja 2020), wtedy kolejnymi falami określano
+niewielkie z perspektywy czasu wzrosty zakażeń. (Tutaj będzie wykres zakażeń z 
+marca 2020).
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 6]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+prosty. Powodów było kilka:
 
 - na stronie ministerstwa publikowane były wyłącznie dane z danego dnia.
 - publicznie dostępne dane dotyczące wielu krajów nie zawierały podziału na
   województwa.
 
+A nas szczególnie interesowały dane właśnie z podziałem na województwa,
+ewentualnie na jeszcze mniejsze jednostki podziału terytorialnego.
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 7]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
 
 # Jak pracowaliśmy z danymi
 
 Te dane musiały zostac sprowadzone do wspólnego formatu, tak aby wszystkie
 stworzone wykresy i analizy można było odtwarzać natychmiast po aktualizacji
-danych. Do tego wykorzystaliśmy pythona i Jupyterlab.
+danych, ponieważ te zmieniały się z dnia na dzień wraz z rozwojem pandemii. Do
+tego wykorzystaliśmy pythona i Jupyterlab.
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 8]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+
+# Nasz cel
+
+Chcemy pokazać to, co udało nam się odkryć w trakcie realizacji pierwszego
+Hackathonu realizowanego w naszym kole, a dotyczył on pandemii. Naszym celem
+było znalezienie, reguł asocjacyjnych i ciekawych zależności w danych z
+zachowań, które docelowo miały nam posłużyć do lepszego zrozumienia sposobu
+rozprzestrzeniania się koronawirusa Sars-Cov-2 i przewidywania ilości
+zachorowań.
```
Comment (pgronkievitz) [inline presentation.md]: this paragraph is a bit long, maybe can be splitted?
it's written `SARS-COV-2` with uppercase as it's abbrev.

File: presentation.md [hunk 9]
```diff
@@ -7,39 +7,88 @@ abstract: |
   Lorem ipsum dolor sit amet
 ---
 
-# Źródła danych
+# O nas
+
+Jesteśmy studentami Politechniki Rzeszowskiej na kierunku Inżynieria i analiza
+danych, a także członkami nowo stworzonego koła uczenia maszynowego.
+[//]: # (Dodać logo koła naukowego)
```
Comment (pgronkievitz) [inline presentation.md:16]: add logo

File: presentation.md [hunk 10]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+niewielkie z perspektywy czasu wzrosty zakażeń. (Tutaj będzie wykres zakażeń z 
+marca 2020).
+
+---
+
+# Jak znaleźć źródła danych?
 
-To było pierwszym sporym wyzwaniem w naszym projekcie, ponieważ szczególnie na
-początku pandemii dostęp do szczegółowych danych nie był tak prosty, ponieważ:
+To było pierwszym pytaniem, jakie zadaliśmy sobie w naszym projekcie, ponieważ
```
Comment (pgronkievitz) [inline presentation.md]: `To było pierwsze pytanie` sounds better.

File: presentation.md [hunk 11]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+szczególnie na początku pandemii dostęp do szczegółowych danych nie był tak
+prosty. Powodów było kilka:
```
Comment (pgronkievitz) [inline presentation.md]: Unnecessarily long sentence - remove `, ponieważ` and replace it with period

File: presentation.md [hunk 12]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
 
 - na stronie ministerstwa publikowane były wyłącznie dane z danego dnia.
 - publicznie dostępne dane dotyczące wielu krajów nie zawierały podziału na
   województwa.
 
+A nas szczególnie interesowały dane właśnie z podziałem na województwa,
```
Comment (pgronkievitz) [inline presentation.md]: `Szczególnie nas interesowały` sounds better imho

File: presentation.md [hunk 13]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+zainteresujemy naszą pracą i dołączą do naszego koła naukowego.
+
+---
+
+# Dane
+
+Jeszcze, gdy zaczynaliśmy, liczba zachorowań nie przekraczała 500 osób
+dziennie (był to okres od marca do maja 2020), wtedy kolejnymi falami określano
+niewielkie z perspektywy czasu wzrosty zakażeń. (Tutaj będzie wykres zakażeń z 
+marca 2020).
```
Comment (pgronkievitz) [inline presentation.md]: insert placeholder instead of comment in parenthesis.

File: presentation.md [hunk 14]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+
+---
+
+# Źródła danych
+
 Na szczęście udało nam się znaleźć inne źródła, tj. strony tworzone przez ludzi,
 którzy codziennie sami zbierali te dane i udostępniali je w formie wykresów.
-Przykładem takiej strony jest koronawirusunas.pl . Dzięki danym na niej zawartym
-udało nam się przeprowadzić.
+Przykładem takiej strony jest koronawirusunas.pl . Dzięki uprzejmości jej
```
Comment (pgronkievitz) [inline presentation.md]: you can create link with `<koronawirusunas.pl>` - also unnecessary space before period

File: presentation.md [hunk 15]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
+twórców udało nam się pobrać dane na niej zawarte i przeprowadzić wszystkie
+analizy.
 
 Do głównych źródeł danych zaliczyć możemy:
 
 - IHME
 - koronawirusunas.pl
 - policja.pl
 
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 16]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
 Główną techniką wykorzystywaną przez nas wył web scraping, stworzyliśmy skrypty,
-które
-(oczywiście za zgodą właścicieli portali) pobierały z nich bieżące dane. Z kolei
-pozostałe dane były udostępnione publicznie w formie skompresowanych plików,
-także aktualizowanych codziennie.
+które (oczywiście za zgodą właścicieli portali) pobierały z nich bieżące dane. Z
+kolei pozostałe dane były udostępnione publicznie w formie skompresowanych
+plików, także aktualizowanych codziennie.
+
+---
```
Comment (pgronkievitz) [inline presentation.md]: unnecessary slide split - this will insert empty one. `#` are creating separate slides

File: presentation.md [hunk 17]
```diff
@@ -7,39 +7,88 @@ abstract: |
...
 
 # Jak pracowaliśmy z danymi
 
 Te dane musiały zostac sprowadzone do wspólnego formatu, tak aby wszystkie
 stworzone wykresy i analizy można było odtwarzać natychmiast po aktualizacji
-danych. Do tego wykorzystaliśmy pythona i Jupyterlab.
+danych, ponieważ te zmieniały się z dnia na dzień wraz z rozwojem pandemii. Do
+tego wykorzystaliśmy pythona i Jupyterlab.
```
Comment (pgronkievitz) [inline presentation.md:77]: `Pythona i Jupyter Lab`

**Review by pgronkievitz — CHANGES_REQUESTED** [review]

File: presentation.md (hunk 9 above)
Comment (finloop) [reply presentation.md:16]: I wrote this comment to indicate that in the presentation, there will be a logo. I treat this as a script to read. The presentation won't have as much text.

**Review by pgronkievitz — APPROVED** [review]

**Review by pgronkievitz — APPROVED** [review]

**Pull request merged**